open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.CONFIG

```diff
@@ -1176,8 +1179,8 @@ PROCEDURE DISPLAYHTML
 
  HTMLVIEWLINE("<td class='lbl width300'>Test accounts list:</td>")
  HTMLVIEWLINE("<td>")
- HTMLVIEWLINE("<input id='testModeAccountList' class='width600' type='text' value='"+
-  SACTESTMODEACCOUNTLIST+"' ")
+ HTMLVIEWLINE("<input id='testModeAccountList' class='width600' type='text'")
```

Comment:
This fix is good as it currently is but just to be safe, should we add a maxlength='109' to the input to limit the user input to 10 account numbers and 9 delimiters between them?

Reply:
Also.... have you checked the other input fields to make sure there are no similar potential issues?

Reply:
added maxlength